support-frontend/assets/pages/new-contributions-landing/components/ContributionFormContainer.jsx

```diff
@@ -172,8 +172,7 @@ function goalReachedTemplate() {
 }
 
 const campaignName = getCampaignName();
-const campaign = campaignName ? campaigns[campaignName] : null;
-
+const campaign = campaignName && campaigns[campaignName] ? campaigns[campaignName] : null;
```

Comment:
not really important, but I think the `campaigns[campaignName]` check is unnecessary because of how `CampaignName` is defined:
`export type CampaignName = $Keys<typeof campaigns>`

Reply:
Good point